gov/src/lib.rs

```diff
@@ -0,0 +1,248 @@
+use std::ffi::OsString;
+use std::fs::read_to_string;
+
+use anyhow::anyhow;
+use ethers::abi::token::{LenientTokenizer, Token, Tokenizer};
+use ethers::abi::AbiParser;
+use ethers::prelude::{Middleware, SignerMiddleware};
+use ethers::types::{Address, U256};
+use regex::Regex;
+
+use std::str::FromStr;
+
+use rad_terminal::args::{Args, Error, Help};
+use rad_terminal::components as term;
+
+use rad_common::ethereum;
+use rad_common::ethereum::{ProviderOptions, SignerOptions};
+
+use crate::governance::Governance;
+
+mod governance;
+
+pub const HELP: Help = Help {
+    name: "gov",
+    description: env!("CARGO_PKG_DESCRIPTION"),
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad gov [<options>...] <command> [<args>...]
+    rad gov [<options>...] vote <proposal-id> (true | false)
+    rad gov [<options>...] propose <proposal-file>
+
+Options
+
+    --help  Print help
+
+Wallet options
+
+    --rpc-url <url>              JSON-RPC URL of Ethereum node (eg. http://localhost:8545)
+    --ledger-hdpath <hdpath>     Account derivation path when using a Ledger hardware device
+    --keystore <file>            Keystore file containing encrypted private key (default: none)
+    --walletconnect              WalletConnect account
```

Comment:
Does this patch rely on the walletconnect one? If so, we need to merge that one first I guess, otherwise you can remove this.